### ISSUE TITLE
"diskful" instead of "diskfull"

### ISSUE THREAD

**Issue by caomengmeng** [xcat2/xcat-core#298]
As @whowutwut  suggested, There are a few other places that also have diskfull with two l . Can you open another issue for this, this is a larger piece of work to clean up all occurrences externally and internally.

So create this issue to track this problem.

**Comment by caomengmeng** [comment]
Hi @whowutwut, do we need to update all related mistakes then this issue can be closed?

**Comment by whowutwut** [comment]
Hi Meng Meng

I think once the pull request is merged we can close this issue. I think we can only do the best and try to correct the document when we see problems. This is just to fix most of the problems to get the doc to a more consistent state. Probably won't fix all occurrences

**Comment by caomengmeng** [comment]
@whowutwut， OK， got it.
Thanks!

**Comment by caomengmeng** [comment]
Close this issue.
